muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/sorter-field.tsx

```diff
@@ -64,7 +64,7 @@ export default class SorterField extends React.Component<SorterFieldProps, Sorte
     if (props.initialValue){
       value = JSON.parse(props.initialValue);
       // We set it up properly
-      items = value.map((v:string)=>this.props.content && this.props.content.items.find(i=>i.id === v));
+      items = value.map((v:string)=>this.props.content && this.props.content.items.find(i=>i.id === v)).filter((v: any) => !!v);
```

Comment:
This is little bit nitpicking, but this line could use some documentation to explain what happens here and why, not only that "we set it up properly". Especially situation like this when there are `any` typing's that doesn't tell anything about what properties objects holds if they even are objects or something else.